src/main/webapp/i18n/de/competency.json

```diff
@@ -174,6 +174,36 @@
             "manage": {
                 "importAllButton": "Alles eines Kurses importieren"
             },
+            "importSettings": {
+                "importRelationsLabel": "Beziehungen importieren",
+                "importRelationsTooltip": "Alle Beziehungen zwischen den importierten Kompetenzen werden ebenfalls importiert",
+                "importExercisesLabel": "Aufgaben importieren",
+                "importExercisesTooltip": "Importiert alle Aufgaben, die mit den importierten Kompetenzen verknüpft sind oder verknüpft sie, wenn eine Aufgabe mit diesem Namen bereits existiert",
+                "importLecturesLabel": "Vorlesungen importieren",
+                "importLecturesTooltip": "Importiert alle Vorlesungen, die mit den importierten Kompetenzen verknüpft sind oder verknüpft sie, wenn eine Vorlesung mit diesem Namen bereits existiert",
+                "referenceDateLabel": "Referenzdatum",
+                "referenceDateTooltip": "Das Referenzdatum wird verwendet, um alle Daten der importierten Resourcen zu setzen. Das früheste Datum dieses Typs wird auf das Referenzdatum gesetzt und alle anderen Daten werden relativ dazu gesetzt.",
+                "referenceDateTypeLabel": "Referenzdatumstyp",
+                "referenceDateTypeTooltip": "'Veröffentlichungsdatum' bezieht sich auf das Veröffentlichungsdatum von Aufgaben und Vorlesungseinheiten sowie das Sichtbarkeitsdatum von Vorlesungen. 'Fälligkeitsdatum' bezieht sich auf das Fälligkeitsdatum von Aufgaben und Kompetenzen sowie Das Enddatum von Vorlesungen.",
+                "referenceDateType": {
+                    "noSelection": "-- Wähle zuerst ein Referenzdatum --",
+                    "releaseDate": "Veröffentlichungsdatum",
+                    "dueDate": "Fälligkeitsdatum"
+                }
+            },
+            "importAll": {
+                "title": "Alle Kompetenzen eines Kurses importieren",
+                "success": "{{ noOfCompetencies }} Kompetenzen aus dem Kurs \"{{ courseTitle }}\" importiert.",
+                "warning": "Es wurden keine Kompetenzen für den Kurs \"{{ courseTitle }}\" gefunden.",
+                "table": {
+                    "title": "Kurse",
+                    "columns": {
+                        "TITLE": "Titel",
+                        "SHORT_NAME": "Kurzname",
```

Comment:
Would it make sense to cross-reference this values to the other i18n entries already existing for these values in the instructor/admin views?

Reply:
In parallel I've already built the competency-management view from scratch again using signals etc. where I'm also gonna clean up the translations